packages/dev/core/src/Meshes/mesh.ts

```diff
@@ -3657,7 +3676,7 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
 
         serializationObject.checkCollisions = this.checkCollisions;
         serializationObject.isBlocker = this.isBlocker;
-        serializationObject.overrideMaterialSideOrientation = this.overrideMaterialSideOrientation;
+        serializationObject.materialSideOrientation = this.sideOrientation;

```

Comment:
Why not simply `sideOrientation`? In fact, side orientation is not really a material property (which is in fact our current problem!) but a geometry property.